packages/terra-form-fieldset/src/Fieldset.module.scss

```diff
@@ -7,7 +7,7 @@
 
   .fieldset-required > .legend:first-child {
     &::before {
-      color: var(--terra-form-fieldset-required-color, #e50000);
+      color: var(--terra-form-fieldset-required-before-color, #e50000);
```

Comment:
nit: I'd rename this to `--terra-form-fieldset-required-legend-before-color`

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/a0ef1a111a15e3ceb49268a954a7c82ccaf332ad